Geometry_Engine/Modify/Close.cs

```diff
@@ -52,6 +52,37 @@ public static Polyline Close(this Polyline polyline, double tolerance = Toleranc
 
             return Create.Polyline(polylinePoints);
         }
+
+        /***************************************************/
+
+        [Description("Close open PolyCurve by adding Line segments between adjecent inner curves not within tolerance. Does not affect Closed curves. Inner Curves of polycurve are assumed to be Ordered such that the end point of inner Curve n should be the start point of Curve n+1 for the curve to be closed.")]
```

Comment:
I started rewriting the description and then realised your last sentence is clear enough. But keeping this comment for reference, please choose the one you like more (in yours there is typo in _adjacent_, just in case).
```suggestion
        [Description("Close open PolyCurve by adding Line segments between each pair of subsequent segment curves if the end point of the first does not lie within tolerance from start point of the second. Does not affect Closed curves.")]
```